bake/bake_test.go

```diff
@@ -1856,3 +1856,37 @@ func TestNetNone(t *testing.T) {
 	require.Len(t, bo["app"].Allow, 0)
 	require.Equal(t, "none", bo["app"].NetworkMode)
 }
+
+func TestVariableValidation(t *testing.T) {
+	fp := File{
+		Name: "docker-bake.hcl",
+		Data: []byte(`
+variable "FOO" {
+  validation {
+    condition = FOO != ""
```

Comment:
Does the condition only allow current variable or links to others(like the value)? If it does allow then could we add tests.

Reply:
Yes it allows access to other variables, I will add additional tests